en/drafts/originals/analyzing-multilingual-text-nltk-spacy-stanza.md

```diff
@@ -24,91 +24,99 @@ doi: XX.XXXXX/phen0000
+Computational text analysis is a broad term that encompasses a wide variety of approaches, methodologies, and Python libraries, all of which can be used to handle and analyze digital texts of all scales. Harnessing computational methods allows you to quickly complete tasks that are far more difficult to perform without these methods. For example, the part-of-speech tagging method described in this lesson can be used to quickly identify all verbs and their associated subjects and objects across a corpus of texts. This could then be used to develop analyses of agency and subjectivity in the corpus (as, for example, in Dennis Tenen's article [Distributed Agency in the Novel](https://muse.jhu.edu/article/898333/summary)). 
 
-In addition to the methods we cover in this lesson, other commonly-performed tasks made easier by computation include sentiment analysis (which provides a quantitative assessment of the sentiment of a text, generally spanning a numerical scale indicating positivity or negativity) and named entity recognition (NER) (which recognizes and classifies entities in a text into categories such as place names, person names, and so on). For further reading on these methods, please see the Programming Historian lessons [Finding Places in Text with the World Historical Gazetteer](https://programminghistorian.org/en/lessons/finding-places-world-historical-gazetteer) and [Corpus Analysis with spaCy](https://programminghistorian.org/en/lessons/corpus-analysis-with-spacy) for named entity recognition, and the lessons [Sentiment Analysis for Exploratory Data Analysis](https://programminghistorian.org/en/lessons/sentiment-analysis) and [Sentiment Analysis with 'syuzhet' using R](https://programminghistorian.org/en/lessons/sentiment-analysis-syuzhet) for sentiment analysis. In addition to the examples above, the lesson  [Introduction to Stylometry with Python](https://programminghistorian.org/en/lessons/introduction-to-stylometry-with-python) may be of interest to those looking to explore additional applications of computational text analysis. This lesson shows how to perform stylometry, a form of exploratory data analysis based on natural language processing, to explore the authorship of texts with disputed authorship. 
+This lesson shows how to perform stylometry, a form of exploratory data analysis based on Natural Language Processing, to explore the authorship of texts with disputed authorship. In addition to the methods we cover in this lesson, other commonly-performed tasks which simplified by computation include sentiment analysis (which provides a quantitative assessment of the sentiment of a text, generally spanning a numerical scale indicating positivity or negativity) and Named Entity Recognition (NER) (which recognizes and classifies entities in a text into categories such as place names, person names, and so on). 
```

Comment:
I'm rather confused by the first sentence here, about stylometry and disputed authorship. I don't find these themes anywhere else in the lesson, so I wonder whether this is truly an accurate description of what you are hoping to teach?